client/landing/stepper/declarative-flow/internals/steps-repository/goals/index.tsx

```diff
@@ -51,12 +53,24 @@ const GoalsStep: Step = ( { navigation } ) => {
 		recordTracksEvent( 'calypso_signup_goals_select', eventProperties );
 	};
 
+	const recordIntentSelectTracksEvent = () => {
+		const hasImportGoal = goals.indexOf( SiteGoal.Import ) > -1;
```

Comment:
I would probably used `.includes()` here - for better readability:

```suggestion
		const hasImportGoal = goals.includes( SiteGoal.Import );
```

Reply:
Thanks! I'm definitely an old school ES5 programmer.